martin/src/srv/cors.rs

```diff
@@ -0,0 +1,243 @@
+use actix_http::Method;
+use log::info;
+use serde::{Deserialize, Serialize};
+
+use crate::{MartinError, MartinResult};
+
+#[derive(thiserror::Error, Debug)]
+pub enum CorsError {
+    #[error("At least one 'origin' must be specified in the 'cors' configuration")]
```

Comment:
we should probably mention the `"*"` as the common value? 

```suggestion
    #[error(r#"At least one 'origin' must be specified in the 'cors' configuration, e.g. origin = "*""#)]
```

Reply:
I think `*` is pretty well known and would pop up in all CORS docs. 
It's also something that should generally be avoided from a security standpoint, so not sure if we should point users to that direction...
If they opt-in into configuring martin's CORS with their custom rules,  they might already have specific origins in mind.